packages/big-design/src/components/Counter/spec.tsx

```diff
@@ -346,15 +363,16 @@ describe('error does not show when invalid type', () => {
       </div>,
     ];
 
-    const { queryByTestId } = render(<FormGroup>{counterMock({ error: errors, ...requiredAttributes })}</FormGroup>);
+    render(<FormGroup>{<CounterMock value={5} error={errors} />}</FormGroup>);
```

Comment:
No need for the `{}`

```suggestion
    render(<FormGroup><CounterMock value={5} error={errors} /></FormGroup>);
```

Reply:
Yeah,  removed. sorry about this.